src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -122,6 +127,15 @@ def probsolve_ivp(
         Whether we want dense output. Optional. Default is ``True``. For the ODE filter,
         dense output requires smoothing, so if ``dense_output`` is False, no smoothing is performed;
         but when it is ``True``, the filter solution is smoothed.
+    diffusion_model : str
+        Which diffusion model to use. The choices are ``'constant'`` and ``'dynamic'``,
+        which implement different styles of
+        online calibration of the underlying diffusion [5]_.
+        Optional. Default is ``'dynamic'``.
+        While we recommend to use correct capitalization for the `diffusion_model` string,
+        upper-case letters will be lower-cased internally.
```

Comment:
sufficiently clear?

Reply:
I think you can also remove the statement about lower-case and upper-case letters.

Reply:
alrighty